### ISSUE TITLE
Skip existing files when transferring

### ISSUE THREAD

**Issue by Temacc0531** [localsend/localsend#443]
### Is your feature request related to a problem? Please describe.

When I transfer a large number of files, some file transfers may fail, I would choose to re-transfer them all directly, but all files will be retransferred

### Describe the solution you'd like

Maybe the files already existing should be automatically skipped

**Comment by LinuxLowell** [comment]
+1. If I am transferring a directory with more than 500 files there is a roughly 50% chance the transfer will fail and I have to start over. If it’s more than 1500 files, it’s pretty much a 100% chance the transfer will fail. “Skip existing files” would be huge.

**Comment by Bugg4** [comment]
+1
This is definitely a needed feature

**Comment by lukpeluk** [comment]
Yeah, please, this is a really needed feature, without this the app doesn't work for big number of files...

I guess it shouldn't be so hard to implement being that skipping files manually is already implemented, and that the app knows if a file already exists because it appends a number to the file name, and I think this is more important than some stuff in the roadmap.

**Comment by Mario-Hofstaetter** [comment]
There is an advanced option for auto-renaming files when receiving, but no "Skip existing" ? 😢

**Comment by tinfoil-hat-net** [comment]
> There is an advanced option for auto-renaming files when receiving, but no "Skip existing" ? 😢

Skip existing would be awesome, I just spend 2 hours copying files and it fails mid term. So I have to delete the file, copy for 2 more hours, just to fail again. Not cool

**Comment by epagone** [comment]
https://github.com/localsend/localsend/issues/2606 is also relevant.

**Comment by rsb-23** [comment]
Please implement this ASAP.
I didn't know this, now I have 7 copies

**Comment by TiagoSimionato** [comment]
I believe it relates to #2478

**Comment by joshhansen** [comment]
This would be incredibly helpful keeping my music collection synced between my laptop and phone. Plus ability to update the file only if the timestamp or hash has changed, like rsync allows.
